andela/coding-geeks-ah-frontend
# #167484550 implemented the design and functionality

#### What does this PR do?
It implements the functionality of viewing and updating a user profile.

#### Description of Task to be done?
1. Adding View Profile Component.
2. Adding View Profile Action.
3. Adding Style for the View Profile Component.
4. Adding Update Profile Component.
5. Adding Update Profile Action.
6. Adding Style for the Update Profile Component.
7. Adding Profile Reducers.

#### How should this be manually tested?
1. Clone the branch.
2. Do **npm install**.
3. Do **npm start**.
4. Login and you will be redirected to your profile. 
5. Check the user profile and try to update it to check the functionality.

#### Screenshots
<img width="1438" alt="Screen Shot 2019-10-16 at 18 37 20" src="https://user-images.githubusercontent.com/34879554/66939796-0a22b580-f044-11e9-99c2-ff56c34fb515.png">
<img width="1440" alt="Screen Shot 2019-10-16 at 18 37 28" src="https://user-images.githubusercontent.com/34879554/66939797-0abb4c00-f044-11e9-8f9a-e4a8ad21c037.png">
